conjur/argument_parser/_hostfactory_parser.py

```diff
@@ -32,11 +32,14 @@ def add_hostfactory_parser(self):
         return self
 
     def _create_hostfactory_parser(self):
+        #Needs TW review
         hostfactory_name = 'hostfactory - Manage hosts and tokens'
```

Comment:
@hzur  - HF tokens? ie 
```suggestion
        hostfactory_name = 'hostfactory - Manage hosts and Host Factory tokens'
```